engine/classes/ElggPluginPackage.php

```diff
@@ -184,7 +187,7 @@ private function validate() {
 			if (!is_readable($this->path . $file)) {
 				$have_req_files = false;
 				$this->errorMsg =
-					_elgg_services()->translator->translate('ElggPluginPackage:InvalidPlugin:MissingFile', [$file]);
+					elgg_echo('ElggPluginPackage:InvalidPlugin:MissingFile', [$file]);
```

Comment:
Is it possible to keep the `elgg_echo` on the same line a `$this->errorMsg`? now that the line is shorter.....